dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/subsystem/FeatureRepository.java

```diff
@@ -621,6 +645,12 @@ private void updateMaps(SubsystemFeatureDefinitionImpl def) {
             if (def.getVisibility() == Visibility.PUBLIC)
                 publicFeatureNameToSymbolicName.put(lowerFeature(cachedAttr.symbolicName), cachedAttr.symbolicName);
 
+            // populate mapping from known, commonly used alternative names to allow hints when the wrong feature
+            // name is specified in a server config.
+            for (String s : cachedAttr.alternateNames) {
+                alternateFeatureNameToPublicName.put(s, cachedAttr.featureName);
```

Comment:
If we do ignore case here then we need to do the following to populate:
`alternateFeatureNameToPublicName.put(lowerFeature(s), cachedAttr.featureName);`

Reply:
I am surprised the FATs pass without making the change to lower case the key here.

Reply:
Will do the lowerFeature call in FeatureDefinitionUtils.getAltNames() instead.